nav-app/src/app/tabs/tabs.component.html

```diff
@@ -1,5 +1,12 @@
 <link href="https://fonts.googleapis.com/css?family=Roboto+Mono:500" rel="stylesheet">
 
+<!-- Banner -->
+<div *ngIf="hasFeature('header') && bannerContent" class="header">
+    <div class="banner">
+        <span role="alert">{{ bannerContent }}</span>
```

Comment:
At present this will sanitize any HTML in the banner, which means we can't put a URL or formatting in the banner. URLs are useful since they would allow use to link to a changelog or release notes for instance.

I believe Angular allows un-santitized HTML via some means (I believe the color gradient code may do this already?), but another option would be to use [ngx-markdown](https://github.com/jfcere/ngx-markdown) to render the banner content allowing us to use markdown formatting in the banner content.